src/cli/main/__main__.py

```diff
@@ -38,7 +38,10 @@
 )
 
 CWD: str = getcwd()
-ON_EXIT: List[Callable[[], None]] = []
+OUT_BASE: str = "/tmp/.makes-out"  # nosec
```

Comment:
let's create this in a less predictable way:

https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp